docs/queries/byrole.mdx

```diff
@@ -15,13 +15,11 @@ import TabItem from '@theme/TabItem'
 getByRole(
   // If you're using `screen`, then skip the container argument:
   container: HTMLElement,
-  role: TextMatch,
+  role: string,
   options?: {
-    exact?: boolean = true,
     hidden?: boolean = false,
     name?: TextMatch,
     description?: TextMatch,
-    normalizer?: NormalizerFn,
     selected?: boolean,
     checked?: boolean,
     pressed?: boolean,
```

Comment:
While I was looking at the PR I noticed that the docs are missing the `suggest` option.
Could we add it here, or should I create a PR for this?

```suggestion
    pressed?: boolean,
    suggest?: boolean,
```

Reply:
can you do this in a separate one? I don't actually know what this option does :D

Reply:
@timdeschryver Do you want me to take it? I have some free time :)

Reply:
Sure, go for it @MatanBobi